docs/streams/developer-guide/processor-api.html

```diff
@@ -343,6 +344,38 @@ <h2>
 </pre></div>
                 </div>
             </div>
+            <div class="section" id="timestamped-state-stores">
+                <span id="streams-developer-guide-state-store-timestamps"></span><h3><a class="toc-backref" href="#id11">Timestamped State Stores</a><a class="headerlink" href="#timestamped-state-stores" title="Permalink to this headline"></a></h3>
+                <p>
+                    Starting in Kafka Streams 2.3, you can store record timestamps in KTables.
+                    A timestamped state store improves stream processing semantics and enables
+                    handling out-of-order data in source KTables, detecting out-of-order joins and aggregations,
+                    and getting the timestamp of the latest update in an Interactive Query.
+                </p>
+                <p>You can query timestamped state stores both with and without a timestamp.</p>
+                <p>
+                    Use <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/TimestampedKeyValueStore.html">TimestampedKeyValueStore</a>
+                    when you need a key-(value/timestamp) store that supports put/get/delete and range queries. 
+                </p>
+                <p>
+                    Use <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/TimestampedWindowStore.html">TimestampedWindowStore</a>
+                    when you need to store windowedKey-(value/timestamp) pairs.</p>
+                <p>
+                    The <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html">Stores</a>
+                    class provides corresponding factory methods for timestamped stores.</p>
+                <p>
```

Comment:
Seems this part belong to section `Defining and creating a State Store` ?

Reply:
Moved these paras into the table.